config/run-tests.sh

```diff
@@ -6,7 +6,7 @@ echo "$LABS"
 
 for lab in $LABS; do
 	echo "Running tests for lab #${lab}"
-
+	export PYTHONPATH=./lab_"${lab}"
```

Comment:
Better export it once at the top of the script so that there is only one export instead of several.